src/applications/gi-sandbox/containers/search/LocationSearchResults.jsx

```diff
@@ -99,19 +100,62 @@ function LocationSearchResults({
     markers.current.push(markerElement);
   };
 
+  const getCurrentLocationCenter = () => {
+    let center = '';
+    if (streetAddress.searchString) {
+      center = new mapboxgl.LngLat(
+        streetAddress.position.longitude,
+        streetAddress.position.latitude,
+      );
+    }
+    return center;
+  };
+
+  const getZoomLevel = distance => {
+    if (distance === '5') {
+      return 11;
+    } else if (distance === '25') {
+      return 10;
+    } else if (distance === '50') {
+      return 9;
+    } else {
+      return 8;
+    }
+  };
+
   useEffect(
     () => {
       markers.current.forEach(marker => marker.remove());
-
+      const currentmarkerElement = document.createElement('div');
+      currentmarkerElement.className = 'current-position';
       if (!map.current || results.length === 0) return; // wait for map to initialize
-
+      if (streetAddress.searchString) {
+        new mapboxgl.Marker(currentmarkerElement)
+          .setLngLat([
+            streetAddress.position.longitude,
+            streetAddress.position.latitude,
+          ])
+          .addTo(map.current);
+      }
       const locationBounds = new mapboxgl.LngLatBounds();
-
       results.forEach((institution, index) => {
         addMapMarker(institution, index, locationBounds);
       });
-
-      map.current.fitBounds(locationBounds, { padding: 20 });
+      if (streetAddress.searchString === autocomplete.location) {
```

Comment:
Use a boolean instead of this because if user changes the search box to a different location then types in their location or selects if from autocomplete the user's location will then appear on the map

- use my location
- drag/type/autocomplete to a new location
- type in user's location <- no marker should appear